### ISSUE TITLE
Check MANIFEST.in

### ISSUE THREAD

**Issue by bersace** [astral-sh/ruff#12066]
Hi,

Thanks for ruff, it's great.

What do you think of implemented [check-manifest](https://github.com/mgedmin/check-manifest) ? This way, we could go all-in with ruff to all linting in Python project. :rocket: 

Regards,
Étienne

**Comment by MichaReiser** [comment]
Yeah, I could see how Ruff does something like this once we have multifile analysis (a good understanding of the dependencies).
